config.ru

```diff
@@ -1,4 +1,10 @@
 # This file is used by Rack-based servers to start the application.
 
+# Don't buffer stdout.  We want logs to be available in real time.
+# See: https://devcenter.heroku.com/articles/logging#writing-to-your-log
+# The rails_stdout_logging also does this, do this can be removed if
+# we start using that.
```

Comment:
This sentence might read better if it said "If we end up using the rails_12factor gem, we can remove this, as it provides this functionality".